bats/fb-content-katello.bats

```diff
@@ -121,9 +121,10 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
-  rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm || true
-  subscription-manager register --force --org="Test_Organization" --activationkey="Test AK" || true
-  subscription-manager status | grep -q "Current"
+  run yum erase -y 'katello-ca-consumer-*'
+  run rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm
+  run subscription-manager register --force --org="Test_Organization" --activationkey="Test AK"
```

Comment:
Will this mask the error and hide it? I'd like to know that both installation of the bootstrap RPM succeeded as well as that registration did not have an error associated with it. While I agree the final condition should be enough to verify the workflow, register is such an important operation that it, in my opinion, should ensure if it errors we see that and it fails the test.